guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -139,6 +139,23 @@ gpgcheck=1
 ----
 endif::[]
 
+ifdef::satellite[]
```

Comment:
Technically, this is not limited to Satellite and can happen on any deployment.

I would however move this down to the `leapp preupgrade` step, and start off with something like:
If `leapp preupgrade` inhibits the upgrade with "Unsupported network configuration" because there are multiple legacy-named network interfaces, please follow the instructions shown by Leapp to rename the interfaces, followed by an installer run to reconfigure {Project} to use the new interface names.

Reply:
For Satellite builds, we might want to link directly to https://access.redhat.com/solutions/4067471, instead of saying "follow the instructions shown by Leapp" ;)